### DISCUSSION TITLE
0.2.51.2-20220206

### DISCUSSION THREAD

**Discussion by eizedev** [eizedev/AirConnect-Synology#53] [Releases]
## AirConnect-Synology - Enhancement Release

[Discussion](https://github.com/eizedev/AirConnect-Synology/discussions/52)

> No new version for DSM6 

> You are new to AirConnect or this is the first time you upgrade from DSM6 to DSM7? Please take a look at the [Documentation and Upgrade Guide](https://github.com/eizedev/AirConnect-Synology#upgrade-from-dsm6-to-dsm7).

### Added 

- Added device `Samsung HW-N950` (#52 ) to default filter
  - The filter will not be updated during upgrade (if you already have AirConnect-Synology installed). All configuration will be kept "as it is" during upgrade process. You need to uninstall and reinstall the package to get back the default configuration with the newly added device. You can of course also add the device manually. Please check the [documentation](https://github.com/eizedev/AirConnect-Synology#airconnectconf) sections for more information.

-------

If you encounter problems with this synology version of AirConnect, please consider opening an [issue here](https://github.com/eizedev/AirConnect-Synology/issues), so i can fix the problem and others can benefit as well.  
If you have any suggestions, ideas or wishes, feel free to open an issue.

## AirConnect

Nothing new, still at `0.2.51.2`

Check the [full CHANGELOG](https://github.com/philippe44/AirConnect/blob/master/CHANGELOG) for more information.

Take the package [suitable for your architecture](https://www.synology.com/en-us/knowledgebase/DSM/tutorial/Compatibility_Peripherals/What_kind_of_CPU_does_my_NAS_have) and check the [readme](https://github.com/eizedev/AirConnect-Synology/blob/master/README.md) if something is unclear.

If you encounter problems with AirConnect in general, please open an [issue](https://github.com/philippe44/AirConnect/issues) at the official repository.

---

**Full Raw Changelog**: https://github.com/eizedev/AirConnect-Synology/compare/0.2.51.2-20220130...0.2.51.2-20220206

<hr /><em>This discussion was created from the release <a href='https://github.com/eizedev/AirConnect-Synology/releases/tag/0.2.51.2-20220206'>0.2.51.2-20220206</a>.</em>

**Comment by lprimak** [comment]
Hi,

I tried to install `AirConnect-dsm7-arm-0.2.51.2-20220206.spk`
on my Synology router RT2600ac with the latest SRM 1.2.5-8227 Update 4 

However, I get "failed to run package service" message with no logs whatsoever.

Can someone help and/or confirm this?
Thank you!

**Reply by eizedev** [reply]
Hi, thanks for reporting this, i will check this in the next days on my router. (I have just updated to Update 4 yesterday).
